openmc/mgxs/mgxs.py

```diff
@@ -1530,21 +1532,21 @@ def rxn_rate_tally(self):
         return self._rxn_rate_tally
 
 
-class NuFissionXS(MGXS):
-    """A fission production multi-group cross section."""
+class FissionXSBase(MGXS):
```

Comment:
About `FissionXS`: Nope... it just wasn't on my radar.  I'll do that one too

About  PEP8: It's pretty funny you brought it up.  The first time I wrote this comment I actually did it with 2 lines... but the second line didn't have much on it so I figured I would just make it one line ;-)

I'll add a line break there...